DirectEmployers/MyJobs
# [PD 2835] responsive css cleanup

Task was to cleanup some of the responsiveness for the css and cleanup some of the code.

To Test:

Pull down the repository. You can shrink the screen size down to mobile and test the tabs and the primary dimensions to see how they work with mobile.
